Makefile

```diff
@@ -8,17 +8,18 @@ help:
 
 WARNINGS=RUSTDOCFLAGS="-D warnings"
 DEBUG_ASSERTIONS=RUSTFLAGS="-C debug-assertions"
+ALL_FEATURES_BUT_ASYNC=--features concurrent,default,std,testing,serde
```

Comment:
I have not found a way to use `--all-features` and exclude a specific feature. 

Hence why we add all features here and simply omit `async`